code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents.dm

```diff
@@ -368,6 +368,34 @@ datum/reagent/slimetoxin
 	description = "A corruptive toxin produced by slimes."
 	color = "#13BC5E" // rgb: 19, 188, 94
 
+datum/reagent/unstableslimetoxin
+	name = "Unstable Mutation Toxin"
+	id = "unstablemutationtoxin"
+	description = "An unstable and unpredictable corruptive toxin produced by slimes."
+	color = "#5EFF3B" //RGB: 94, 255, 59
+	metabolization_rate = INFINITY //So it instantly removes all of itself
+
+datum/reagent/unstableslimetoxin/on_mob_life(var/mob/living/carbon/human/H as mob)
+	..()
+	H << "<span class='warning'><b>You crumple in agony as your flesh wildly morphs into new forms!</b></span>"
+	H.visible_message("<b>[H]</b> falls to the ground and screams as their skin bubbles and froths!") //'froths' sounds painful when used with SKIN.
+	H.Weaken(3)
+	sleep(30)
```

Comment:
Noone, I'm pointing the finger to vent my frustration from the shit code I have to suffer through.